Modules/Devices/src/Devices.Application/Identities/Queries/GetRegisteredIdentityDetails/GetRegisteredIdentityDetailsQuery.cs

```diff
@@ -0,0 +1,6 @@
+﻿using MediatR;
+
+namespace Backbone.Modules.Devices.Application.Identities.Queries.GetRegisteredIdentityDetails;
+public class GetRegisteredIdentityDetailsQuery : IRequest<GetRegisteredIdentityDetailsResponse>
```

Comment:
```suggestion
public class GetOwnIdentityQuery : IRequest<GetOwnIdentityResponse>
```

Do this renaming everywhere it applies, please

Reply:
Changes made.